bears/python/VultureBear.py

```diff
@@ -1,25 +1,45 @@
-import re
 from shutil import which
-
+from vulture import Item, Vulture
```

Comment:
I'd sort the imports alphabetically, i.e. coalib, dependency_management, vulture